### DISCUSSION TITLE
发送图片

### DISCUSSION THREAD

**Discussion by JingYiWeiRan** [foxwhite25/qq.py#9] [Q&A] [answered]
怎么用qq.py库发送本地图片

**Comment by foxwhite25** [comment, marked as answer]
使用file传参，
https://qqpy.readthedocs.io/zh-cn/latest/api.html#qq.abc.Messageable.send

JingYiWeiRan ***@***.***> 于 2023年10月24日周二 下午9:49写道：

**Reply by JingYiWeiRan** [reply]
谢谢！！解决了
